package/MDAnalysis/coordinates/base.py

```diff
@@ -980,6 +979,78 @@ def __repr__(self):
             nframes=self.n_frames,
             natoms=self.n_atoms
         ))
+    
+    def timeseries(self, asel: Optional['AtomGroup']=None,
+                   start: Optional[int]=None, stop: Optional[int]=None,
+                   step: Optional[int]=None,
+                   order: Optional[str]='fac') -> np.ndarray:
+        """Return a subset of coordinate data for an AtomGroup
+
+        Parameters
+        ----------
+        asel : AtomGroup (optional)
```

Comment:
I know we're just doing the same as what memoryreader does, but it just struck me that `asel` is such a weird parameter name for an atomgroup. Not much we can do here, but if I'm not the only one maybe we can consider changing this for 3.0 (in a different PR, let's get this merged!).

Reply:
I'll raise an issue

Reply:
Raised in #3911